docs/index.rst

```diff
@@ -67,26 +67,26 @@ Example Usage
 
 ::
 
+    from locasticsearch import Locasticsearch
     from datetime import datetime
-    from elasticsearch import Elasticsearch
-    es = Elasticsearch()
+
+    es = Locasticsearch()
 
     doc = {
-        'author': 'kimchy',
-        'text': 'Elasticsearch: cool. bonsai cool.',
-        'timestamp': datetime.now(),
+        "author": "kimchy",
+        "text": "Elasticsearch: cool. bonsai cool.",
+        "timestamp": datetime(2010, 10, 10, 10, 10, 10),
     }
-    res = es.index(index="test-index", id=1, body=doc)
-    print(res['result'])
+    res = es.index(index="test-index", doc_type="tweet", id=1, body=doc)
 
-    res = es.get(index="test-index", id=1)
-    print(res['_source'])
+    res = es.get(index="test-index", doc_type="tweet", id=1)
```

Comment:
With 7.x we shouldn't use `doc_types` in examples, why are we using this here?

Reply:
I was also confused. Stripped them away now